apps/sasquatch/src/main/res/values/strings.xml

```diff
@@ -29,6 +29,13 @@
     <string name="key" tools:ignore="MissingTranslation">Key</string>
     <string name="value" tools:ignore="MissingTranslation">Value</string>
     <string name="type" tools:ignore="MissingTranslation">Type</string>
+    <string-array name="typed_property_types" tools:ignore="MissingTranslation">
+        <item>Boolean</item>
+        <item>Double</item>
+        <item>Long</item>
+        <item>Date/Time</item>
```

Comment:
Is "Date/Time" intentional? Not "DateTime"?

Reply:
It's just a test app, let's use something that's closest to an actual type - I think `DateTime` is good

Reply:
It's copied from the custom properties. If we decide to change it, it will be better if it is consistent